github-arc/runner/values-kubernetes.yaml

```diff
@@ -0,0 +1,37 @@
+githubConfigUrl: "https://github.com/utilitywarehouse"
+
+githubConfigSecret: default
+
+maxRunners: 5
+minRunners: 1
+
+runnerScaleSetName: "cluster-os-kubernetes"
```

Comment:
This translates to a name here: https://github.com/organizations/utilitywarehouse/settings/actions/runners

Our current runners have this label in `exp-1-merit`: `uw-infra-exp` - whats the best way to set this for different envs?

Reply:
i'll try patching from the kustomize in kubernetes-manifests. will leave to just this for now until there is a requirement to deploy multiple runnerSets in a single namespace otherwise we will have to rename a lot more 

```
  - target:
      kind: AutoscalingRunnerSet
      name: cluster-os-kubernetes
    patch: |-
      - op: replace
        path: "/spec/runnerScaleSetName"
        value: exp-1-merit
```